lib/jekyll-sass-converter/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module JekyllSassConverter
-  VERSION = "1.5.2"
+  VERSION = "1.6.0"
```

Comment:
This will certainly require a major version bump, but Jekyll 3.x will not load jekyll-sass 2.x